internal/test/rhp/v2/rhp.go

```diff
@@ -13,10 +13,15 @@ import (
 	"sort"
 	"time"
 
-	rhpv2 "go.sia.tech/core/rhp/v2"
+	rhp2 "go.sia.tech/core/rhp/v2"
```

Comment:
renaming this feels unnecessary 🤔

Reply:
I think it's better to be consistent, if only for the sake of easily copy pasting between files. `hostd` aliases it as `rhp2` all the time, the `rhpv2` came from `renterd` I suppose. I don't mind changing it though if you disagree.

Reply:
I agree, let’s keep it consistent

Reply:
ah, ok, I mistakenly assumed it was `rhpv2` everywhere since that's what I'm used to seeing in my own code. 😅 Agreed, consistency is important.